catalog.html

```diff
@@ -55,6 +55,151 @@
         </ul>
       </nav>
     </header>
+    <main class="main-inner">
+      <h1 class="visually-hidden">Каталог</h1>
+      <section class="catalog-breadcrumb">
+        <h2>Моноподы для селфи</h2>
+        <ul class="catalog-breadcrumb-list">
+          <li>
+            <a class="catalog-breadcrumb-link" href="#">Главная</a>
+          </li>
+          <li>
+            <a class="catalog-breadcrumb-link" href="#">Каталог товаров</a>
+          </li>
+          <li>
+            <a class="catalog-breadcrumb-link" href="#">Моноподы для селфи</a>
+          </li>
+        </ul>
+      </section>
+      <section class="catalog-filter">
+        <h2>ФИЛЬТР:</h2>
```

Comment:
Все заглавные делаются через стили, здесь же нужно писать по простому:
```suggestion
        <h2>Фильтр:</h2>
```